### ISSUE TITLE
Backend Client not outputting thumbnail url

### ISSUE THREAD

**Issue by mchatman** [edgestorejs/edgestore#50]
The thumbnail url is not included in the response when using the backend client following the instructions [here](https://edgestore.dev/docs/backend-client#upload-a-blob)

```
import { backendClient } from "@/lib/edgestore/edgestore-server";
import { unstable_noStore } from "next/cache";

export async function POST(req) {
  unstable_noStore();

  const formData = await request.formData();
  const file = formData.get("file");
  const type = formData.get("type");
  const r = await backendClient.publicImages.upload({
    content: {
      blob: new Blob([file]),
      extension: type
    }
  });
}
```

Confirmed that I am using a image bucket which should include the thumbnail image.

```import { initEdgeStore } from '@edgestore/server';
import { createEdgeStoreNextHandler } from '@edgestore/server/adapters/next/app';
import { initEdgeStoreClient } from '@edgestore/server/core';

const es = initEdgeStore.create();

const edgeStoreRouter = es.router({
  publicImages: es.imageBucket(),
});

export const handler = createEdgeStoreNextHandler({
  router: edgeStoreRouter,
});

export const backendClient = initEdgeStoreClient({
  router: edgeStoreRouter,
});
```

Example output:

```
{
  url: 'https://files.edgestore.dev/z9ye2vb42uvksfhp/publicImages/_public/19173981-a217-41d1-ab49-b4fae951104c.jpg',
  size: 496304,
  metadata: {},
  path: {},
  pathOrder: []
}
```

**Comment by perfectbase** [comment]
@mchatman Thank you for reporting it.
It might be missing on the backend client.
Will take a look as soon as I can.

**Comment by perfectbase** [comment]
@mchatman This should be fixed on the latest version (0.1.7).
Let me know if you have any problems with it.
Thank you again for reporting it! 🙏 
Closing the issue.
